piston/data_processor.py

```diff
@@ -80,7 +80,7 @@ def _validate_not_empty(self, data_item, key, item):
     def _process_file_data_field(self, data, files, key, data_item):
         try:
             filename = data_item.get('filename')
-            file_content = cStringIO(base64.b64decode(data_item.get('content')).decode('utf-8'))
+            file_content = BytesIO(base64.b64decode(data_item.get('content').encode('utf-8')))
```

Comment:
V Pythonu 3 b64decode bere i vrací byte a ne str, takže to takhle nefungovalo...